evap/results/templates/results_widget_range_slider.html

```diff
@@ -0,0 +1,12 @@
+<section class="range-slider">
+    <div class="row mt-3">
+        <span class="col-4 text-start text-secondary"></span>
+        <span class="col-4 range-values"></span>
+        <span class="col-4 text-end text-secondary"></span>
+    </div>
+    <input class="form-range" name="low" value="" min="" max="" step="1"
+           type="range">
+
+    <input class="form-range" name="high" value="" min="" max="" step="1"
+           type="range">
```

Comment:
Wrapping at 80 seems a bit excessive to me when our python line length is 120. Can we just

```suggestion
    <input class="form-range" name="low" value="" min="" max="" step="1" type="range" />
    <input class="form-range" name="high" value="" min="" max="" step="1" type="range" />
```